keyboards/rs60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,48 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QWERTY,
+  _HYPER,
+};
+
+#define KC_ KC_TRNS
+#define _______ KC_TRNS
```

Comment:
QMK defines this keycode alias automatically. This line can be deleted.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

```suggestion
```